### DISCUSSION TITLE
Impact of sensor location - foot vs ankle

### DISCUSSION THREAD

**Discussion by rakeshpilkar82** [mad-lab-fau/gaitmap#56] [Q&A] [answered]
Hi,
I am wondering about the impact on the accuracy of derived measures if the algorithm is applied to the data recorded from the ankle location (above lateral malleolus) compared to the foot location. As the raw trajectories of these two locations differ, would it significantly impact the event detection outcome?

**Comment by AKuederle** [comment, marked as answer]
Hi great question! All algorithms are only validated for sensors at the foot. However, some algorithms are likely to work for an ankle position as well.
I have not validated or tested any of that, but here a couple of assumptions:

- Trajectory reconstruction should work the same. Potentially even better as the ankle position has more reliable points of 0-velocity
- DTW based stride segmentation will not work at all (at least not with the default template). It might be possible to get reasonable results with a custom template
- Similar for HMM based segmentation. The default model will likely not work, but a custom re-trained model might provide reasonable results
- Event detection: IC/HS will liekly work. Toe-off is unlikely to work.
- Final Parameter calculation: all temporal parameters that don't depend on the toeoff should owrk reasonably well. Main spatial paramters like stride length and gait velocity should also work. However, all angle based parameters will loose their meaning as the orientation of an ankle sensor is independent of the actual sole angle of the foot.

**Comment by rakeshpilkar82** [comment]
Thank you for your quick response. I will get back if I have further questions.


Rakesh Pilkar, PhD
Senior Data Scientist


[signature_3431492910]
Pioneering the Digital Transformation of Clinical Research™


 [Image]   (850) 495 9503
[signature_197540266]  ***@***.******@***.***>
[Image]  70 N Baylen St, Suite 400. Pensacola, FL 32502<https://maps.google.com/?q=70%20N%20Baylen%20St,%20Suite%20400.%20Pensacola,%20FL%2032502>
[signature_2375159752]<https://www.linkedin.com/company/actigraph/mycompany/>  [Image] <https://www.facebook.com/actigraphcorp>   [Image] <https://twitter.com/actigraph>

[Image]<https://hubs.la/Q022XMKh0>



From: Arne Küderle ***@***.***>
Date: Tuesday, January 16, 2024 at 4:06 PM
To: mad-lab-fau/gaitmap ***@***.***>
Cc: Rakesh Pilkar ***@***.***>, Author ***@***.***>
Subject: Re: [mad-lab-fau/gaitmap] Impact of sensor location - foot vs ankle (Discussion #56)

Hi great question! All algorithms are only validated for sensors at the foot. However, some algorithms are likely to work for an ankle position as well.
I have not validated or tested any of that, but here a couple of assumptions:

  *   Trajectory reconstruction should work the same. Potentially even better as the ankle position has more reliable points of 0-velocity
  *   DTW based stride segmentation will not work at all (at least not with the default template). It might be possible to get reasonable results with a custom template
  *   Similar for HMM based segmentation. The default model will likely not work, but a custom re-trained model might provide reasonable results
  *   Event detection: IC/HS will liekly work. Toe-off is unlikely to work.
  *   Final Parameter calculation: all temporal parameters that don't depend on the toeoff should owrk reasonably well. Main spatial paramters like stride length and gait velocity should also work. However, all angle based parameters will loose their meaning as the orientation of an ankle sensor is independent of the actual sole angle of the foot.
